calour/database.py

```diff
@@ -149,31 +149,34 @@ def enrichment(exp: Experiment, features, dbname, *args, **kwargs):
 
     Returns
     -------
-    pandas.DataFrame
+        pandas.DataFrame with  info about significantly enriched terms. The columns include:
 
-        The dataframe contains info about significantly enriched terms. The columns include:
+            feature : str the feature
```

Comment:
make it a list? like one comments of a previous PR:

   * feature: xx
   * pval: xx